src/operator/drop.ts

```diff
@@ -0,0 +1,20 @@
+import { Subscribable, Listener, Subscription } from '../interfaces'
+
+export class DropOperator<T> implements Subscribable<T> {
+  constructor (private amount: number, public _subscribable: Subscribable<T>) {}
+
+  subscribe (listener: Listener<T>): Subscription {
+    return this._subscribable.subscribe(new DropListener<T>(this.amount, listener))
+  }
```

Comment:
Do you think it's possible to merge DropOperator and DropListener? Having them separate is useful in RxJS v5 where we have unicast by default, but considering that after we "multicast" using `xs.create(subscribable)`, from that point onwards we can assume hot only. I can't remember but I suspect that having these two merged in one is better for perf.